tests/EpisodeIntegrationServiceTests/ReceiveDataTests/ReceiveDataTests.cs

```diff
@@ -275,4 +334,5 @@ public async Task ReceiveData_ShouldLogErrorAndReturnIfUrlsAreNotConfigured()
             (Func<object, Exception, string>)It.IsAny<object>()),
             Times.Once);
     }
+
```

Comment:
I think we're missing some tests that cover that rowIndex, successCount and failureCount are being logged and the logs are what we expect

Reply:
Unit test on this bit has been designed and will be implemented after the date logic merges, which will be used as the test data.